core/src/test/java/org/elasticsearch/cluster/MinimumMasterNodesIT.java

```diff
@@ -132,7 +132,9 @@ public void testSimpleMinimumMasterNodes() throws Exception {
         });
         state = client().admin().cluster().prepareState().setLocal(true).execute().actionGet().getState();
         assertThat(state.blocks().hasGlobalBlock(DiscoverySettings.NO_MASTER_BLOCK_ID), equalTo(true));
-        assertThat(state.nodes().getSize(), equalTo(1)); // verify that we still see the local node in the cluster state
+        // verify that both nodes still in the cluster state but no master
```

Comment:
Nit: `both nodes still` -> `both nodes are still` and `but no master` -> `but there is no master`